app/Http/Controllers/OrderController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\CheckoutRequest;
+use App\Models\Product;
+use App\Models\Order;
+use Session;
+use Auth;
+
+class OrderController extends Controller
+{
+    public function getCheckout()
```

Comment:
```suggestion
    public function getListItemsInCart()
```
`getCheckout` có nghĩa gì ko em? (tat)